.scrutinizer.yml

```diff
@@ -127,6 +127,11 @@ build_failure_conditions:
     - 'project.metric("scrutinizer.quality", < 6)'
 
 build:
+    environment:
+        php:
+            version: 7.0.20
```

Comment:
Would it benefit us to just specify the PHP version as `7.0`? I assume that's possible.